homework/list.js

```diff
@@ -1,79 +1,85 @@
 'use strict';
 
-(function() {
+function List() {
+	this._listNoteContent = [];
+	this._wrapper = document.querySelector('#note-list-wrapper');
+}
 
-  var _listNoteContent = [];
-  var _wrapper = document.querySelector('#note-list-wrapper');
+List.prototype = {
```

Comment:
You can re-write the chain as

    this.fetchList().then(this.updateList.bind(this))
                          .then(this.drawList.bind(this))

It's more concise.